ui/lib/pki/addon/templates/keys/index.hbs

```diff
@@ -0,0 +1,63 @@
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @params={{array "keys.import"}} @type="download">
+      Import
+    </ToolbarLink>
+    <ToolbarLink @params={{array "keys.generate"}} @type="add">
+      Generate
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+<p class="has-padding">Below is information about the private keys used by the issuers to sign certificates. While
+  certificates represent a public assertion of an identity, private keys represent the private part of that identity, a
+  secret used to prove who they are and who they trust.</p>
+{{#if (gt this.model.length 0)}}
+  {{#each this.model as |model|}}
+    <LinkedBlock class="list-item-row" @params={{array "roles.role.details" model.id}} @linkPrefix={{this.mountPoint}}>
+      <div class="level is-mobile">
+        <div class="level-left">
+          <div>
+            <Icon @name="certificate" class="has-text-grey-light" />
+            <span class="has-text-weight-semibold is-underline">
+              {{model.id}}
+            </span>
+            <div class="is-flex-row has-left-margin-l has-top-margin-xs">
+              {{#if model.isDefault}}
```

Comment:
Curious, where does this attribute come from? I noticed it on the model, but not in the docs. I may be missing context, but could we instead check for `if (eq model.keyName "default)`

Reply:
It comes from the LIST network request. So similar to how you had to clients serializer it gets added to the model from what is returned on the LIST endpoint. In this view we only have the model data built out from what is returned on the LIST endpoint. It's not the full model for the key, so keyName isn't available to us. When we go to the edit/read/update views we'll then have the model fully built out for the key. 

Here is the LIST response.
![image](https://user-images.githubusercontent.com/6618863/191380607-337a9f45-74f0-432f-8291-d2898a978fd1.png)

Reply:
As an aside: the payload returned is slightly different than the docs. I have a question out about that. E.g. it shows the param is_default when the payload on the docs does not show that.

Reply:
Thank you for the screenshot! Super helpful. This makes sense now